FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -124,6 +124,53 @@ The browser verifies it witnessed a Fetch request to the `seller`’s origin wit
 
 ## Device Orchestrated Multi-Seller Auctions
 
+Bidding and Auction Services auctions can also be used as component auctions. It
+is possible for multiple Bidding and Auction Services auctions to be used as
+component auctions for an on-device top-level auction.
+
+For efficiency, a top-level seller can construct encrypted requests for multiple
```

Comment:
Will `sellers` use be detectable or must one catch the exception?

```
Uncaught TypeError: Failed to execute 'getInterestGroupAdAuctionData' on 'Navigator': Failed to read the 'seller' property from 'AdAuctionDataConfig': Required member is undefined.
```

Reply:
Yeah, I think catching the exception is probably the best way to handle it.